Apps/Src/SendTritium.c

```diff
@@ -388,7 +388,7 @@ static void ReverseDriveHandler(){
         UpdateDisplay_SetGear(DISP_REVERSE);
     }
     velocitySetpoint = -MAX_VELOCITY;
-    currentSetpoint = percentToFloat(map(accelPedalPercent, ACCEL_PEDAL_THRESHOLD, PEDAL_MAX, CURRENT_SP_MIN, CURRENT_SP_MAX));
+    currentSetpoint = percentToFloat(map(accelPedalPercent, ACCEL_PEDAL_THRESHOLD, PEDAL_MAX, CURRENT_SP_MIN, CURRENT_SP_MAX * 0.2)); // Limit max motor speed to 20% in reverse 
```

Comment:
This won't actually directly limit the speed to 20%, but limit the current. You might need to keep it at 100% current until the speed limit is reached, and then completely cut the current (or quickly drop the limit to zero).